source/mongodump.txt

```diff
@@ -279,16 +281,18 @@ Data Exclusion
 :binary:`~bin.mongodump` excludes the content of the ``local`` database
 in its output.
 
-:binary:`~bin.mongodump` output only captures the documents in the
+:binary:`~bin.mongodump` output only exports the documents in the
```

Comment:
I think it could be unclear to users what we mean when we say we do not export index data. I'm worried they may think we don't export the index definitions either. Perhaps we could say "...does not include index data, only the index definitions".

Reply:
Rewrote.

:binary:`~bin.mongodump` output:

- Contains the documents in the database and the index definitions.
- Does not contain the index data.